api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb

```diff
@@ -18,6 +18,7 @@ angular.module('mnoEnterprise.configuration', [])
   .constant('ADMIN_PANEL_CONFIG', <%= Hash(Settings.admin_panel).to_json %>)
   .constant('PAYMENT_CONFIG', <%= Hash(Settings.payment).to_json %>)
   .constant('ORGANIZATION_MANAGEMENT', <%= Hash(Settings.organization_management).to_json %>)
+  .constant('SUB_TENANT_CONFIG', <%= Hash(Settings.sub_tenant).to_json %>)
```

Comment:
Could we nest that under `ADMIN_PANEL_CONFIG`?
That's easier to manage that'll be easier for v2 migration.